src/detect-smb-filename.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2017 Open Information Security Foundation
```

Comment:
2022

Reply:
Done in https://github.com/OISF/suricata/pull/7011/commits/196fb7326a0cd333f2e204afbc87666bdc470474